elements/rh-disclosure/rh-disclosure.ts

```diff
@@ -141,11 +136,19 @@ export class RhDisclosure extends LitElement {
     this.summaryEl.focus();
   }
 
-  #mutationsCallback(): void {
-    const hasJumpLinks: RhJumpLinks | null = this.querySelector('rh-jump-links');
-    if (hasJumpLinks && !this.classList.contains('has-jump-links')) {
-      this.classList.add('has-jump-links');
-    }
+  #handleJumpLinkChildren(event: Event) {
+    const sheet: CSSStyleSheet = new CSSStyleSheet();
```

Comment:
do this at the top level at the module, so that we're only parsing the css string once, but adopt the same stylesheet object here in the callback, so we don't have to recreate it each time.